modules/appeals_api/app/models/appeals_api/appellant.rb

```diff
@@ -165,9 +165,8 @@ def address
       form_data['address'] || {}
     end
 
-    def claimant_headers_present?
-      form_data.dig('data', 'attributes', 'claimant', 'lastName').present? ||
-        auth_headers.include?('X-VA-NonVeteranClaimant-Last-Name')
+    def claimant_details_present?
+      form_data['lastName'].present? || auth_headers.include?('X-VA-NonVeteranClaimant-Last-Name')
```

Comment:
As I was making the tests for `Appellant` more robust (because of the `delegate` changes I made), I found this mistake 😬